std/file.d

```diff
@@ -3590,6 +3593,63 @@ else version (DragonFlyBSD)
 
         return buffer.assumeUnique;
     }
+    else version (OpenBSD)
+    {
+        import core.sys.posix.unistd : getpid;
+        import std.conv : to;
+        import std.exception : enforce, errnoEnforce;
+        enum
+        {
+            CTL_KERN = 1,
+            KERN_PROC_ARGS = 55,
+            KERN_PROC_ARGV = 1,
+        }
```

Comment:
why redefine when they could be reused from `core.sys.openbsd.sys.sysctl`?

Reply:
This function is not very well separated since someone added `thisExePathHasSysCtlProcName`, I'd rather tackle all platforms at once in a follow-up PR.